code/modules/clothing/glasses/hud.dm

```diff
@@ -8,7 +8,7 @@
 	name = "\improper HealthMate HUD"
 	desc = "A heads-up display that scans the humans in view and provides accurate data about their health status."
 	icon_state = "healthhud"
-	deactive_state = "degoggles"
+	deactive_state = "healthhud_d"
```

Comment:
Do not rename the degoggles icon state, it is used extensively for other HUDs as well